index.html

```diff
@@ -169,5 +172,163 @@ <h3 class="mg-sm-top-bottom">Combining Avatar Classes</h3>
         class="avatar avatar-xlg avatar-outline-warning"
```

Comment:
not sure which tag is this. Looks like a typo.

Reply:
I didn't understand your review properly.  I don't see any typo. Can you elaborate?